### DISCUSSION TITLE
Was there a change to the search (label:) syntax? None of my links are working as of this morning

### DISCUSSION THREAD

**Discussion by shannonmccleary-ihm** [community/community#53316] [Pull Requests]
### Select Topic Area

Bug

### Body

I have had bookmark shortcuts setup and working for the last 2 years that allow me to see PRs with a certain set of labels. As of this morning none of my shortcuts are working. For instance we have one pull request with a status of `REVIEW_REQUESTED` currently, the filter string below does not return it

Was working, but not working anymore:
`is:open is:pr label:REVIEW_REQUESTED label:QA_PASSED_READY_TO_MERGE label:UAT_PASSED_READY_TO_MERGE`

Was working, now not working:
`is:open is:pr label:REVIEW_REQUESTED,QA_PASSED_READY_TO_MERGE,UAT_PASSED_READY_TO_MERGE`

This works however, but wont work for us
`is:open is:pr label:REVIEW_REQUESTED`

Myself and 8 other devs are experiencing this. What changed? It also appears that wrong PRS are being pulled in that do not match the filter labels: ![Screenshot 2023-04-20 095050](https://user-images.githubusercontent.com/60754534/233404186-bec8b5a4-a274-4f30-a8ca-b6169c2ebac7.png)

**Comment by maxcleme** [comment]
I'm experiencing the same behaviour. If I remove one label from one issue and then search for issues with the label, it would still appears in the search result (but without the label) for quite some time. Looks like indexing issue to me.
